partcad/src/partcad/user_config.py

```
@@ -10,143 +10,110 @@
 from pathlib import Path
 import shutil
-import yaml
+import vyper
 
 from . import logging as pc_logging
 
 
-class UserConfig:
+class UserConfig(vyper.Vyper):
     @staticmethod
     def get_config_dir():
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `get_config_dir`

(ANN205)